### PR TITLE
Add account type

### PR DESCRIPTION
**Pull request by klaasnicolaas**
This PR  will do:
- ✨ Add the account type
- 📚 Update the documentation
- 🔨 Update naming in example code

### CODE REVIEW

File: forecast_solar/models.py [hunk 1]
```diff
@@ -25,6 +25,14 @@ def _timed_value(at: datetime, data: dict[datetime, int]) -> int | None:
     return None
 
 
+class AccountType(str):
```
Comment (MartinHjelmare) [inline forecast_solar/models.py:28]:
```suggestion
class AccountType(str, Enum):
```

File: forecast_solar/models.py [hunk 2]
```diff
@@ -38,13 +46,23 @@ class Estimate:
...
+    api_rate_limit: int
     api_timezone: str
 
     @property
     def timezone(self) -> str:
         """Return API timezone information."""
         return self.api_timezone
 
+    @property
+    def account_type(self) -> int:
```
Comment (ludeeus) [inline forecast_solar/models.py:58]:
```suggestion
    def account_type(self) -> AccountType:
```

File: forecast_solar/models.py [hunk 3]
```diff
@@ -38,13 +47,23 @@ class Estimate:
...
     def timezone(self) -> str:
         """Return API timezone information."""
         return self.api_timezone
 
+    @property
+    def account_type(self) -> AccountType:
+        """Return API account_type information."""
+        if self.api_rate_limit == 60:
+            return AccountType.PERSONAL
+        elif self.api_rate_limit == 5:
```
Comment (ludeeus) [inline forecast_solar/models.py:63]:
```suggestion
        if self.api_rate_limit == 5:
```

`elif` is not needed here, it matches the first and returns, or it gets here anyway

**Comment by klaasnicolaas** [comment]
I think we can merge it now? 😄

**Review by MartinHjelmare — APPROVED** [review]

**Pull request merged**